test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,214 @@
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.ok;
+				node.expect(res).to.have.property('transactionId').to.equal(transaction.id);
+				done();
+			});
+			goodTransactions.push(transaction);
+		});
+
+		it('using positive overflown amount should fail', function (done) {
+			var account = node.randomAccount();
+			var transaction = node.lisk.transaction.createTransaction(account.address, 1298231812939123812939123912939123912931823912931823912903182309123912830123981283012931283910231203, node.gAccount.password);
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.have.string('Invalid transaction body - Failed to validate transaction schema: Value 1.2982318129391238e+99 is greater than maximum 10000000000000000');
+				done();
+			});
+			wrongTransactions.push(transaction);
+		});
+
+		it('using negative overflown amount should fail', function (done) {
+			var account = node.randomAccount();
+			var transaction = node.lisk.transaction.createTransaction(account.address, -1298231812939123812939123912939123912931823912931823912903182309123912830123981283012931283910231203, node.gAccount.password);
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.have.string('Invalid transaction body - Failed to validate transaction schema: Value -1.2982318129391238e+99 is less than minimum 0');
+				done();
+			});
+			wrongTransactions.push(transaction);
+		});
+	});
+
+	describe('processing', function () {
+		it('when sender has no funds should fail', function (done) {
+			var noFundsAccount = node.randomAccount();
+			var transaction = node.lisk.transaction.createTransaction('1L', 1, noFundsAccount.password);
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.match(/Account does not have enough LSK: [0-9]+L balance: 0/);
+				done();
+			});
+			wrongTransactions.push(transaction);
+		});
+
+		it('using entire balance should fail', function (done) {
+			var account = node.randomAccount();
+			var transaction2 = node.lisk.transaction.createTransaction(account.address, Math.floor(node.gAccount.balance), node.gAccount.password);
+			sendTransaction(transaction2, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.match(/^Account does not have enough LSK:/);
+				done();
+			});
+			wrongTransactions.push(transaction2);
+		});
+
+		it('when sender has funds should be ok', function (done) {
+			var account = node.randomAccount();
+			var transaction = node.lisk.transaction.createTransaction(account.address, 1, node.gAccount.password);
```

Comment:
so why not creating more transactions from multisignature and secondPasshprase accounts here (and everywhere where a transaction is created)?